sycl/source/detail/windows_pi.cpp

```diff
@@ -13,29 +13,17 @@
 #include <windows.h>
 #include <winreg.h>
 
+#include "win_proxy_loader.hpp"
+
 namespace sycl {
 __SYCL_INLINE_VER_NAMESPACE(_V1) {
 namespace detail {
 namespace pi {
 
 void *loadOsLibrary(const std::string &PluginPath) {
```

Comment:
Since the function is used for loading plugins only, can we rename it to something like `loadPlugin`?